http://sourceware.org/ml/gdb-cvs/2012-09/msg00082.html

### src/gdb/ChangeLog	2012/09/17 07:15:47	1.14664
### src/gdb/ChangeLog	2012/09/17 07:26:54	1.14665
## -1,3 +1,8 @@
+2012-09-17  Siddhesh Poyarekar  <siddhesh@redhat.com>
+
+	* infrun.c (restore_infcall_suspend_state): Eliminate single-use
+	variable LEN.
+
 2012-09-17  Jan Kratochvil  <jan.kratochvil@redhat.com>
 
 	PR 14119
--- src/gdb/infrun.c	2012/09/17 07:09:34	1.558
+++ src/gdb/infrun.c	2012/09/17 07:26:55	1.559
@@ -6777,11 +6777,10 @@
   if (inf_state->siginfo_gdbarch == gdbarch)
     {
       struct type *type = gdbarch_get_siginfo_type (gdbarch);
-      size_t len = TYPE_LENGTH (type);
 
       /* Errors ignored.  */
       target_write (&current_target, TARGET_OBJECT_SIGNAL_INFO, NULL,
-		    inf_state->siginfo_data, 0, len);
+		    inf_state->siginfo_data, 0, TYPE_LENGTH (type));
     }
 
   /* The inferior can be gone if the user types "print exit(0)"
